### PR TITLE
Create nmap.aliases.bash

### PR DESCRIPTION
**Pull request by Niil78**

### CODE REVIEW

File: nmap.aliases.bash [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Niil78
```
Comment (nwinkler) [inline nmap.aliases.bash]: We typically don't have author information in the files

File: nmap.aliases.bash [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#2019
```
Comment (nwinkler) [inline nmap.aliases.bash]: No need to have the year here either...

File: nmap.aliases.bash [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+cite 'about-alias'
+about-alias 'Nmap scanner'
+
+alias Nmap0='echo "Escaner normal"; sudo nmap' #Escaner Basico
```
Comment (nwinkler) [inline nmap.aliases.bash]: Can you please convert the comments and `echo` output to English language? We have an international audience with Bash-it...

File: nmap.aliases.bash (hunk 2 above)
Comment (Niil78) [reply nmap.aliases.bash]: Sorry. It is my first contribution.
I really made the aliases for me. But I wanted to upload them here.

File: nmap.aliases.bash (hunk 3 above)
Comment (Niil78) [reply nmap.aliases.bash]: If I'm going to do the whole sheet. I want to do a meno like in pacman.
:D

**Comment by nwinkler** [comment]
Thanks for the update! There are a couple of general comments:

* There's still a lot of Spanish language in the aliases (the "menu" alias and the `echo` statements). Is this really required? I'm asking since the rest of Bash-it is exclusively English language. I understand that it might be convenient for you to see things in Spanish, but this might not be a good fit for the rest of the Bash-it community.
* The structure of the aliases with the `xxx0` being a kind of help/menu and then the other aliases with incremented counters does not really match the rest of Bash-it's aliases, where you typically use a somewhat descriptive name for the alias, e.g. `nmappara` for the paranoid version.

Would you mind addressing these?

**Comment by Niil78** [comment]
What u need?

**Comment by nwinkler** [comment]
@Niil78 Sorry, not sure I understand your question. What do you mean?

**Comment by Niil78** [comment]
what need for this public (sorry 4 my inglish)

**Comment by nwinkler** [comment]
Sorry, don't know what you mean with "public"...

**Comment by Niil78** [comment]
the code its end. i eliminate all spanish lang

**Comment by nwinkler** [comment]
Well, like I said above, the general structure of the aliases (with the 0, 1, 2 at the end) does not match the other aliases in Bash-it, which use a descriptive name or abbreviation (like `nmappara` instead of `nmap3`). I really don't want to introduce another paradigm for naming aliases - this will be confusing for users.

**Pull request closed without merging**